### ISSUE TITLE
Telephone number form group on user details page is missing error class

### ISSUE THREAD

**Issue by stuartmccoll** [heritagefund/funding-frontend#237]
The wrapping `div` element with a `class` attribute of `govuk-form-group` in `views/user/details/show.html.erb` is missing the conditional logic that we use to append the `govuk-form-group--error` class when the model's `errors` hash has an attribute which matches this field.

**Comment by stuartmccoll** [comment]
Closing this - the conditional logic exists, but we don't perform any validation on the phone number field at the moment.
